ui/src/main/webapp/lib/graphql-errors/index.js

```diff
@@ -15,7 +15,16 @@ const friendlyMessage = {
   INVALID_URL_ERROR: 'Invalid URL.',
   NO_EXISTING_CONFIG: 'The configuration does not exist on the server.',
   INVALID_URI_ERROR: 'Invalid URI.',
-  UNAUTHORIZED: 'Unauthorized. A valid username & password may be required to connect.'
+  UNAUTHORIZED: 'Unauthorized. A valid username & password may be required to connect.',
+  CANNOT_BIND: 'Cannot authenticate user.',
+  INVALID_DN: 'The distinguished name is invalid.',
```

Comment:
✏️ Should probably be clearer that the distinguished name is incorrectly formatted.
'The distinguished name has an invalid format.'